components/ActionBarImage.js

```diff
@@ -1,11 +1,14 @@
 import React from 'react'
 import { View, Image } from 'react-native'
+import logo from '../assets/react.png'
 import TeadsSampleAppBlack from '../assets/Teads-Sample-App.imageset/Teads-Sample-App-black.png'
+import styles from '../styles/styles'
```

Comment:
Mmmmmhh, not sure centralizing every style in one single file is good idea 😅 i would but them in the same file for every component